src/pages/home/owners/OwnersTable.tsx

```diff
@@ -1,166 +1,196 @@
 import theme from "theme";
-import { Dispatch, SetStateAction, useEffect, useState } from "react";
-import { Typography } from "elements";
-import { Owner, useWindowDimensions } from "common";
-import PendingBadge from "assets/images/PendingBadge";
-import { TablePagination } from "components";
+import { useGetCollaboratorsQuery } from "modules/song";
+import { TableSkeleton, Typography } from "elements";
+import { useWindowDimensions } from "common";
+import { TableCell, TableHeadCell, TablePagination } from "components";
+import NoOwnersYet from "./NoOwnersYet";
+
 interface OwnersTableProps {
-  ownersData: Owner[];
-  page: number;
-  setPage: Dispatch<SetStateAction<number>>;
+  query: string;
+  totalCollaborators: number;
 }
 
 export default function OwnersTable({
-  ownersData,
-  page,
-  setPage,
+  query,
+  totalCollaborators,
 }: OwnersTableProps) {
-  const rowHeight = 60;
   const headerHeight = 245;
-  const footerHeight = 65;
+  const footerHeight = 40;
   const bottomPadding = 30;
-  const [rowsPerPage, setRowsPerPage] = useState(0);
-  // Used to avoid a layout jump when reaching the last page with empty rows.
-  const emptyRows =
-    page > 1 ? Math.max(0, page * rowsPerPage - ownersData.length) : 0;
+  const rowHeight = 65;
+  const viewportWidth = useWindowDimensions()?.width;
+  const viewportHeight = useWindowDimensions()?.height;
+  const [rowsPerPage, setRowsPerPage] = useState(1);
+  const [page, setPage] = useState(1);
+
+  let collaboratorsToRequest = rowsPerPage;
+
   const lastRowOnPage = (page - 1) * rowsPerPage + rowsPerPage;
-  const StyledTableCell = styled(TableCell)<TableCellProps>`
-    border-color: ${theme.colors.grey500};
-    color: ${theme.colors.grey100};
-    height: ${rowHeight}px;
-  `;
-  const StyledTableCellHeader = styled(TableCell)<TableCellProps>`
-    border-color: ${theme.colors.grey500};
-    color: ${theme.colors.grey100};
-    font-weight: ${theme.typography.fontWeightBold};
-  `;
+  const totalPagesCount = Math.ceil(totalCollaborators / rowsPerPage);
+  const remainingSongsOnLastPage = totalCollaborators % rowsPerPage;
 
-  // determines how many rows to display per page
-  const windowHeight = useWindowDimensions()?.height;
-  useEffect(() => {
-    setRowsPerPage(
-      windowHeight
-        ? Math.floor(
-            (windowHeight - headerHeight - footerHeight - bottomPadding) /
-              rowHeight
-          )
-        : 5
-    );
-  }, [windowHeight]);
+  // Determines how many collaborators to request for the last page
+  if (page === totalPagesCount) {
+    collaboratorsToRequest =
+      remainingSongsOnLastPage > 0 ? remainingSongsOnLastPage : rowsPerPage;
+  }
+
+  const {
+    data: collaboratorsData = [],
+    isLoading,
+    isSuccess,
+  } = useGetCollaboratorsQuery({
+    offset: page - 1,
+    limit: collaboratorsToRequest,
+    phrase: query,
+  });
 
   const handlePageChange = (
-    event: React.ChangeEvent<unknown>,
+    _event: React.ChangeEvent<unknown>,
```

Comment:
Is this common convention to include the parameter name? I thought usually only the underscore `_` is present if the parameter isn't utilized/a throwaway? Or is this just more descriptive and better convention, especially with TypeScript?

Reply:
Correct, the underscore is used as a convention to indicate that the parameter is intentionally unused or ignored, which is what is happening in this case.